Automattic/wp-calypso
# Share Site: Adjust copy based on feedback

Fixes https://github.com/Automattic/dotcom-forge/issues/2593

## Proposed Changes

Adjusts the copy used in the Share Site for Preview components, and adds an inline support link:


### Sites Page Modal

<img width="649" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/73e05c04-cea1-4c3d-bf2a-040b27e4c484">

<img width="634" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/ed274795-d268-4ef7-9ffc-18b83c6fa04b">

### Privacy Settings, Launched

<img width="788" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/58551db8-3605-4b1c-bf6c-aeee7f3c8c43">


<img width="781" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/5794cacd-7b53-4661-9a9a-b607b3e5a285">

### Privacy Settings, Unlaunched

<img width="805" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/c4b68139-961b-4685-bee3-8ea4876af942">


<img width="817" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/0a493584-d1f9-400b-9267-7e4a8d0af96d">


## Testing Instructions

1. Create a new WordPress.com Business site.
2. Navigate to Sites, click on the ellipsis menu for your new site, click on 'Share site for preview'
3. Verify the modal appears and works as expected.
4. Navigate to General Settings for the site.
5. Verify the share site component appears as expected.
